rest/codegen/paths.go

```diff
@@ -2,24 +2,57 @@ package codegen
 
 import (
 	"fmt"
-	"strings"
 	"text/template"
 
 	"goa.design/goa.v2/codegen"
 	goadesign "goa.design/goa.v2/design"
 	"goa.design/goa.v2/rest/design"
 )
 
+// note: conversion array to string path https://play.golang.org/p/0QHmyJeFhR
+
 const pathT = `{{range $i, $route := .Routes}}
```

Comment:
isn't the initial newline going to cause extra blank lines in the generated code?